src/Components/Card.css

```diff
@@ -0,0 +1,7 @@
+.h3, h3 {
+   font-family: 'Pinyon Script', cursive;
+}
+
+.h5, h5 {
+   font-family: 'Quattrocento', serif;
+}
```

Comment:
One good habit to build is leaving a newline at the end of your files. It doesn't usually cause any problems for css or js files, but can create issues if you're working in something like python, or if you're using something that concatenates your code.